apps/src/templates/progress/progressTestHelpers.js

```diff
@@ -0,0 +1,26 @@
+import _ from 'lodash';
+import { LevelStatus } from '@cdo/apps/util/sharedConstants';
+import { createStore } from 'redux';
+import Immutable from 'immutable';
+
+export const fakeLesson = (name, id, lockable=false) => ({name, id, lockable});
```

Comment:
Moved this out of `apps/test`. Had some debate about where they best belong, but ultimately landed on putting them here since it seemed less weird to have tests require helpers from src than the reverse.